src/meta/src/rpc/metrics.rs

```diff
@@ -734,7 +758,16 @@ pub async fn start_fragment_info_monitor<S: MetaStore>(
                                 let table_id_str = table.id.to_string();
                                 meta_metrics
                                     .table_info
-                                    .with_label_values(&[&table_id_str, &actor_id_str, &table.name])
+                                    .with_label_values(&[
+                                        &mv_id_str,
+                                        &table_id_str,
+                                        &actor_id_str,
+                                        &table.name,
+                                    ])
+                                    .set(1);
+                                meta_metrics
+                                    .mv_info
```

Comment:
`mv_info` gives more useful information than `table_info` so I think we can just keep `mv_info`. 
Suggestions: 
- Remove table_info metric and its grafana panel
- Add a new column in actor_info to store the state table id associated with the actor if any.

Reply:
Agree.

Reply:
>Add a new column in actor_info to store the state table id associated with the actor if any.

As a state table will be shared by multiple actors, let's just remove `table_info` panels.